tests/Commands/PurgeCommandTest.php

```diff
@@ -119,6 +119,7 @@ private function setUpMocks()
      * @param $line
      *
      * @return array
+     * @throws \ReflectionException
```

Comment:
Can't you do the new notation of `$this->throwsException()`?

Reply:
This is just for the IDE. The other was phpunit 8 dropping annotations.